raiden-ts/src/channels/state.ts

```diff
@@ -24,7 +24,7 @@ const _ChannelEnd = t.readonly(
     withdraw: UInt(32),
     locks: t.readonlyArray(Lock),
     balanceProof: Signed(BalanceProof),
-    withdrawRequests: t.readonlyArray(Signed(WithdrawRequest)),
+    pendingWithdraws: t.readonlyArray(t.union([Signed(WithdrawRequest), Signed(WithdrawExpired)])),
```

Comment:
`withdrawRequests` before contained only the request part of the withdraw flow. Now it contains both, because we need to store everything to retry. Actually, I'll need to add also `WithdrawConfirmation` to it, as per what we've talked in the call, but the idea is that the array holds the series of messages related to a request (request, confirmation, expired), and these messages get cleared when `WithdrawExpired` is `Processed` or when the withdraw is detected on-chain (the happy case). The renaming was to make it more obvious that this is not only requests anymore.